content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md

```diff
@@ -2,143 +2,337 @@
-CLI single gesture, thanks to [the `@pulumi/eks`
-package](https://github.com/pulumi/pulumi-eks). Let's see how.
+no easy task. AWS now offers EKS to ease a lot of that burden -- but
+it's still difficult to get up and running. Pulumi's infrastructure as
+code to the rescue! We can now provision an entire EKS cluster with a
+single CLI command, thanks to [the `@pulumi/eks`
+package](https://github.com/pulumi/pulumi-eks). Let's see how.
 <!--more-->
 
+TODO: update this image?
 ![Pulumi making EKS Easy](./easy-eks.png)
 
+{{% notes type="info" %}}
+*Update (December 2020)*
+
+Our popular EKS library was developed
+back when Pulumi only supported TypeScript. Since that time, we've added support
+for additional languages, and have received frequent requests to bring EKS support
+to them as well. Today, we're excited to announce that EKS is available for Python,
+and support for .NET and Go is coming soon!
+{{% /notes %}}
+
 To get started, download the free and open source
-[Pulumi SDK]({{< relref "/docs/get-started/install" >}}). The SDK includes
-the CLI we'll be using below, and requires AWS credentials to access
+[Pulumi SDK]({{< relref "/docs/get-started/install" >}}). The SDK includes
+the CLI we'll be using below, and requires AWS credentials to access
 your AWS account and provision resources. If you already have the AWS
 CLI installed and configured, you're already all set to go. Otherwise,
 [check out the docs here]({{< relref "/docs/intro/cloud-providers/aws/setup" >}})
 to set things up.
 
-From here, there are two ways to proceed:
+From here, there are two ways to proceed:
 
 - Provision a new cluster with a single CLI command
 - Use infrastructure as code to manage clusters ... including Kubernetes resources themselves!
 
 ## Installing Amazon EKS with Pulumi
 
-With the Pulumi CLI installed and configured, a single command will
+TODO: is this still true now that we have multi-lang support?
+
+With the Pulumi CLI installed and configured, a single command will
 provision your cluster:
 
     $ pulumi new https://github.com/pulumi/apps/eks
 
-Using this template will walk you through configuring a few settings
-such as the region you'd like to deploy into, the
-[Amazon Elastic Compute Cloud](http://aws.amazon.com/ec2) (Amazon EC2)
-instance type to deploy into the cluster, the cluster size, and whether to
-install the Kubernetes Dashboard into your cluster automatically.
+Using this template will walk you through configuring a few settings
+such as the region you'd like to deploy into, the instance type, and
+number of cluster nodes.
 
-The CLI will first show you a preview of the AWS and Kubernetes
+The CLI will first show you a preview of the AWS and Kubernetes
 resources that will be deployed to create the cluster, before actually
 performing any operation. Once you have reviewed and selected "Yes," the
 cluster will begin deploying. The terminal output will be something like
 the following:
 
+TODO: may want to update this image
```

Comment:
I think this particular one is probably okay - there's not much language-specific here.

Reply:
Agreed, I think it adds value